src/Data/Functor/Plus.hs

```diff
@@ -120,6 +126,11 @@ instance Plus Seq where
   zero = mempty
 #endif
 
+#ifdef MIN_VERSION_unordered_containers
+instance (Hashable k, Eq k) => Plus (HashMap k) where
```

Comment:
Isn't this `Eq k` constraint unnecessary?

Reply:
Remember that `Plus` is a subclass of `Alt`. The `Eq` is necessary for the `Alt` implementation, so is necessary here.

Reply:
Afraid not: https://hackage.haskell.org/package/unordered-containers-0.2.10.0/docs/Data-HashMap-Lazy.html#v:union

Reply:
Oops, I forgot about the superclass. Never mind!